lib/moog-require.js

```diff
@@ -7,6 +7,30 @@ const importFresh = require('import-fresh');
 const resolveFrom = require('resolve-from');
 const regExpQuote = require('regexp-quote');
 
+const getDependencies = ({
```

Comment:
Use `function` and place these at the end, don't front-load the file with implementation detail code that makes it harder to read the overall intent.